db/db_impl/db_impl.h

```diff
@@ -1216,6 +1216,9 @@ class DBImpl : public DB {
   InstrumentedMutex trace_mutex_;
   BlockCacheTracer block_cache_tracer_;
 
+  // constant false canceled flag, used when the compaction is not manual
+  const std::atomic<bool> canceled_always_false_{false};
```

Comment:
nit: Rename the const to the format of `kSomeThing`. How about `kManualCompactionCanceledFalse`?